doc/rados/operations/health-checks.rst

```diff
@@ -1015,12 +1015,12 @@ told to roll back to a previous version of the object. See
 SLOW_OPS
 ________
 
-One or more OSD requests is taking a long time to process.  This can
+One or more OSD or monitor requests is taking a long time to process.  This can
 be an indication of extreme load, a slow storage device, or a software
 bug.
 
-The request queue on the OSD(s) in question can be queried with the
-following command, executed from the OSD host::
+The request queue on the daemon in question can be queried with the
```

Comment:
s/on the daemon/for the daemon/

I think in this case, "for" is the preposition that fits better.

Reply:
@zdover23 updated and repushed.